packages/server/Dockerfile

```diff
@@ -82,6 +82,8 @@ EXPOSE 4001
 # due to this causing yarn to stop installing dev dependencies
 # which are actually needed to get this environment up and running
 ENV NODE_ENV=production
+# this is required for isolated-vm to work on Node 20+
+ENV NODE_OPTIONS="--no-node-snapshot"
```

Comment:
Its probably worth applying these same changes to the `worker` as well - JS isn't really used in it, but it could be used in regards to email templates etc - theres nothing stopping it from happening so should support it.